# AwesomeGenomics
Cancer Data Science's go to place for excellent genomics tools and packages

__Here is a list of lists of awesome bio-informatics tools!__
- https://github.com/danielecook/Awesome-Bioinformatics :A curated list of awesome Bioinformatics libraries and software.
- https://github.com/WooGenome/awesome-bioinformatics: A curated list of awesome Bioinformatics databases, softwares, libraries, toolboxes, pipelines, books, courses, tutorials and more.
- https://github.com/mikelove/awesome-multi-omics: List of software packages for multi-omics analysis
- https://github.com/seandavi/awesome-single-cell: Community-curated list of software packages and data resources for single-cell, including RNA-seq, ATAC-seq, etc.
- https://github.com/crazyhottommy/ChIP-seq-analysis: A curated list of awesome ChIP-seq things


- https://github.com/j-andrews7/awesome-bioinformatics-benchmarks: A curated list of bioinformatics bench-marking papers and resources.


__Awesome Bio reads__
- https://github.com/zhongmicai/awesomeBiology: awesome biology
- https://github.com/raivivek/awesome-biology: Curated list of resources for Biology.
- https://github.com/XindiWu/Awesome-Machine-Learning-in-Biomedical-Healthcare-Imaging: A list of awesome selected resources towards the application of machine learning in Biomedical/Healthcare Imaging, inspired by
- https://github.com/yangkky/Machine-learning-for-proteins: Listing of papers about machine learning for proteins.
- https://github.com/gokceneraslan/awesome-deepbio: A curated list of awesome deep learning applications in the field of computational biology

__Others__

- https://github.com/GuanLab/Awesome-Bioinformatics: 2018 Recommended Papers to Read in Bioinformatics as Voted by Bioinformaticians
- https://github.com/keller-mark/awesome-biological-visualizations: A list of web-based interactive biological visualizations.
- https://github.com/hussius/deeplearning-biology: A list of deep learning implementations in biology
- https://github.com/caufieldjh/awesome-bioie: ðŸ§« A curated list of resources relevant to doing Biomedical Information Extraction (including BioNLP)
 

## Computational

### Worklow Management

- [Terra](https://app.terra.bio/): _a very low barrier to entry, workflow and data management platform for medical and research genomics_
- Nextflow
- Google Genomics Pipelines
- reflow
- snakemake

### python Basics


### R Basics


### Neat python

- [POT](https://github.com/rflamary/POT): library for solving optimal transport optimization problems.
- [Itrask](https://github.com/iamtrask): set of differential privacy tools for analyzing data!
- [JKBio](): Jeremie Kalfon's python scripts for genomics.
- [CDSpy](): some plotting scripts for genomic analysis.



### Neat R


### Other

## Genomics

### Mutations

- __[Bash]__ [Mutect1](https://github.com/broadinstitute/mutect): _an awesome cancer mutation caller, especially for calling point mutations)

- __[py]__ [

### Expression


#### Differential Expression



### Others



## Single Cell

### Expression


#### Differential Expression


## Epigenomics

A lot is available for ChIPseq from [crazyhottommy's repo](https://github.com/crazyhottommy/ChIP-seq-analysis) 

### ChIPseq and related

- [EPIC2](https://github.com/biocore-ntnu/epic2): calling peaks from ChIP seq
- [DeepBind](http://tools.genes.toronto.edu/deepbind) predicting binding location from previous binding data with CNN
- [coda](https://github.com/kundajelab/coda): denoising ChIPseq data with CNNs
- [CREAM](https://github.com/bhklab/CREAM): identifying clusters of functional regions within the genome from ChIPseq data
- [ngsplot](https://github.com/shenlab-sinai/ngsplot): multi omics viz tool at specific locus
- [epi-corr](https://github.com/broadinstitute/epi-correlation): correlation tool for pairs of ChIP seq data

#### diff binding 

- [MACS2 diff binding](https://github.com/taoliu/MACS/wiki/Call-differential-binding-events): how to do differential binding analysis with MACS2.


### ATACseq


### HiCseq and related
