# AwesomeGenomics
Cancer Data Science's go to place for excellent genomics tools and packages

_If something needs to be changed or added, feel free to create a pull request._



## Awesome awesomeness
### Here is a list of lists of awesome bio-informatics tools!
- https://github.com/danielecook/Awesome-Bioinformatics :A curated list of awesome Bioinformatics libraries and software.
- https://github.com/WooGenome/awesome-bioinformatics: A curated list of awesome Bioinformatics databases, softwares, libraries, toolboxes, pipelines, books, courses, tutorials and more.
- https://github.com/mikelove/awesome-multi-omics: List of software packages for multi-omics analysis
- https://github.com/seandavi/awesome-single-cell: Community-curated list of software packages and data resources for single-cell, including RNA-seq, ATAC-seq, etc.
- https://github.com/crazyhottommy/ChIP-seq-analysis: A curated list of awesome ChIP-seq things


- https://github.com/j-andrews7/awesome-bioinformatics-benchmarks: A curated list of bioinformatics bench-marking papers and resources.


### Awesome Bio reads
- https://github.com/zhongmicai/awesomeBiology: awesome biology
- https://github.com/raivivek/awesome-biology: Curated list of resources for Biology.
- https://github.com/XindiWu/Awesome-Machine-Learning-in-Biomedical-Healthcare-Imaging: A list of awesome selected resources towards the application of machine learning in Biomedical/Healthcare Imaging, inspired by
- https://github.com/yangkky/Machine-learning-for-proteins: Listing of papers about machine learning for proteins.
- https://github.com/gokceneraslan/awesome-deepbio: A curated list of awesome deep learning applications in the field of computational biology

__Others__

- https://github.com/GuanLab/Awesome-Bioinformatics: 2018 Recommended Papers to Read in Bioinformatics as Voted by Bioinformaticians
- https://github.com/keller-mark/awesome-biological-visualizations: A list of web-based interactive biological visualizations.
- https://github.com/hussius/deeplearning-biology: A list of deep learning implementations in biology
- https://github.com/caufieldjh/awesome-bioie: ðŸ§« A curated list of resources relevant to doing Biomedical Information Extraction (including BioNLP)
- https://github.com/mahmoud/awesome-python-applications: awesome apps
- https://github.com/serhii-londar/open-source-mac-os-apps: awesome macapps
- https://github.com/shenwei356/awesome: awesome datascience
- https://github.com/krzjoa/awesome-python-data-science: awesome datascience2!
- https://github.com/lukasz-madon/awesome-remote-job: awesome remote job
- https://github.com/xiamx/awesome-sentiment-analysis: sentiment analysis
- https://github.com/analyticalmonk/awesome-neuroscience: awesome neuroscience
- https://github.com/xhacker/awesome-github-extensions: improve your github experience
 

### Awesome python
- [awesome notebooks](https://github.com/firmai/awesome-google-colab)
- [awesome notebooks2](https://github.com/nma/awesome-ipython)
- [awesome python](https://github.com/vinta/awesome-python)
- [awesome python2!](https://github.com/uhub/awesome-python)
- [awesome python3!!](https://github.com/trananhkma/fucking-awesome-python)
- [awesome python security](https://github.com/guardrailsio/awesome-python-security)
- [everythings pytorch](https://github.com/bharathgs/Awesome-pytorch-list)
- [goto snippets](https://github.com/progrmoiz/python-snippets)
- [wanna do more async?](https://github.com/timofurrer/awesome-asyncio)
- [scrap the web](https://github.com/lorien/awesome-web-scraping)
- [python chemistry](https://github.com/lmmentel/awesome-python-chemistry)
- [decorate your code](https://github.com/lord63/awesome-python-decorator)
- [put your notebook to the next phase](https://github.com/markusschanta/awesome-jupyter)
- [...and your pandas](https://github.com/tommyod/awesome-pandas)


## Computational

### Worklow Management

- [Terra](https://app.terra.bio/): _a very low barrier to entry, workflow and data management platform for medical and research genomics_
	- [Dalmatian](https://github.com/broadinstitute/dalmatian): to interact with Terra in python
- Nextflow
- Google Genomics Pipelines
- reflow
- snakemake

### Dataset Management

- Terra
- GS
- [Taiga](https://github.com/broadinstitute/taiga):
	- [taigaR](https://github.com/broadinstitute/taigr)
	- [taigaPy]()


### bash Basics


### python Basics

- [bokeh](https://github.com/bokeh/bokeh/blob/master/CHANGELOG): Best Interactive Plot with JS

### R Basics



### Neat python

- [POT](https://github.com/rflamary/POT): library for solving optimal transport optimization problems.
- [Itrask](https://github.com/iamtrask): set of differential privacy tools for analyzing data!
- [JKBio](): Jeremie Kalfon's python scripts for genomics.
- [CDSpy](): some plotting scripts for genomic analysis.
- [Selene](https://github.com/FunctionLab/selene): a framework for training sequence-level deep learning networks
- [nbstripout](https://github.com/fastai/fastai-nbstripout): removes notebook outputs before git pushing
- [voila](https://github.com/voila-dashboards/voila): turns your jupyter notebook into awesome web apps
- [ngrock](https://ngrok.com/): secure introspectable tunnels: transforms `http://8.8.8.8:8888` into `https://www.ngrock.id.com`
- [SublimeJEDI](https://github.com/srusskih/SublimeJEDI): python+sublime
- [functional-python](https://github.com/sfermigier/awesome-functional-python): learn!
- [python parser](https://parsy.readthedocs.io/en/latest/overview.html)
- [plot in your terminal](https://github.com/daleroberts/itermplot)

### Neat R



### Neat Bash


### Other

- [Nice tools and Discussion on DL](https://github.com/lexfridman/mit-deep-learning): Tutorials, assignments, and competitions for MIT Deep Learning related courses. https://deeplearning.mit.edu
- [Kipoi](https://github.com/kipoi/models): model Zoo for DL in genomics!
- [interpretability](https://github.com/jphall663/awesome-machine-learning-interpretability): start building interpretable models

## Genomics

### read mapper

- [NextGenMap](https://github.com/Cibiv/NextGenMap): NextGenMap is a flexible highly sensitive short read mapping tool that handles much higher mismatch rates than comparable algorithms 
- [bwa]
- [bowtie]
- []
- []
-

### Mutations

- [Mutect1](https://github.com/broadinstitute/mutect): _an awesome cancer mutation caller, especially for calling point mutations)
- [Lancet](https://github.com/nygenome/lancet): Based on microAssembly with a decision model
- __[R]__ [ACE](https://github.com/tgac-vumc/ACE): Absolute CN estimation from low coverage WGS
- [Absolute](https://github.com/absolute-community/absolute): Absolute CN estimation from WES/WGS giving off many predictions to choose from (need prior knowledge)
	- [DoAbsolute](https://github.com/ShixiangWang/DoAbsolute): an R package to Automate the Absolute algorithm
- [Strelka](https://github.com/Illumina/strelka): small variant caller (germline/somatic)
- [Manta](https://github.com/Illumina/manta): the SV caller version of Strelka
- [DeepVariant](https://github.com/google/deepvariant): Fast Variant Calling with DL


#### Effect Prediction

- __[py]__ [DeepSea](https://github.com/danvk/deepsea): Prediction of Effect of non coding variant on ChIP seq binding/Expression/.. with DL
- __[py]__ [SpliceAI](https://github.com/Illumina/SpliceAI): Predicting Effect of Variant on the splicing (modeling the spliceosome) with DL
- __[py]__ [ExPecto](https://github.com/FunctionLab/ExPecto): Predicting effect of NC variant on Expression with DL


#### Annotators

- [OncoKB](https://github.com/oncokb/oncokb-annotator): annotates MAF from oncoKB DB
- [Oncotator](https://github.com/broadinstitute/oncotator): annotates MAF from many DBs (not very well documented)
-

### Expression


- [STAR-Fusion](https://github.com/STAR-Fusion/STAR-Fusion/wiki): call fusions from RNAseq data



#### Next gen Expression

- [slamdunk](https://github.com/t-neumann/slamdunk): to analyse slamseq data
 - [JK/slamdunk](https://github.com/jkobject/slamdunk): Paired End version

#### Differential Expression



### Others



## Single Cell

some competitors: __[py]__ [awesome single cell](https://github.com/seandavi/awesome-single-cell)

### Expression

- [novosparc](https://github.com/rajewsky-lab/novosparc): reconstruct 3D disposition from scRNAseq and some known location+expression atlases



#### Differential Expression



## Epigenomics

A lot is available for ChIPseq from [crazyhottommy's repo](https://github.com/crazyhottommy/ChIP-seq-analysis) 

- [pyGenomeTracks](https://github.com/deeptools/pyGenomeTracks): viewer/plotter for Multi-Epigenomics data

### ChIPseq and related

- [MACS2](https://github.com/taoliu/MACS): go to peak caller
- [EPIC2](https://github.com/biocore-ntnu/epic2): calling peaks from ChIP seq
- [RSEG](https://github.com/smithlabcode/rseg): another peak caller
- [coda](https://github.com/kundajelab/coda): denoising ChIPseq data with CNNs
- [CREAM](https://github.com/bhklab/CREAM): identifying clusters of functional regions within the genome from ChIPseq data
- [ngsplot](https://github.com/shenlab-sinai/ngsplot): multi omics viz tool at specific locus
- [epi-corr](https://github.com/broadinstitute/epi-correlation): correlation tool for pairs of ChIP seq data
- [SUPERmerge](https://github.com/Bohdan-Khomtchouk/SUPERmerge): a ChIP-seq read pileup analysis and annotation algorithm for investigating alignment (BAM) files of diffuse histone modification ChIP-seq datasets with broad chromatin domains at a single base pair resolution level
- [nf-core/chipseq](https://github.com/nf-core/chipseq): Complete ChIPseq pipeline on nextflow
- [pyBigWig](https://github.com/deeptools/pyBigWig): interacts with bigwig from python
- [deepTools](https://github.com/deeptools/deepTools): a set of cmd line tools for epigenomics data 
- [EnrichedHeatmap](https://github.com/jokergoo/EnrichedHeatmap): make the famous enrichment at locus heatmap plots.

#### diff binding 

- [MACS2 diff binding](https://github.com/taoliu/MACS/wiki/Call-differential-binding-events): how to do differential binding analysis with MACS2.


#### Predictors

- [DeepBind](http://tools.genes.toronto.edu/deepbind): predicting binding location from previous binding data with CNN
	-[DeeperBind](https://github.com/hassanzadeh/DeeperBind): a deeper version
- [Basenji](https://github.com/calico/basenji): Predicts Binding from Mutations with CNN
- [ABC model](https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction): Predicts Enhancer-gene links

### ATACseq



### HiCseq and related

- [HiCExplorer](https://github.com/deeptools/HiCExplorer): process, normalize and visualize HiC data 
